tern/analyze/common.py

```diff
@@ -555,9 +556,42 @@ def get_git_sha(path_to_toplevel):
     return sha_info
 
 
+def get_git_url(path_to_toplevel):
+    '''Given a absolute path to a git repository, return the project url.'''
+    command = ['git', 'remote', '-v']
+    url_info = '(not found)'
+    try:
+        output = subprocess.check_output(  # nosec
+            command, stderr=subprocess.DEVNULL, cwd=path_to_toplevel)
+        if isinstance(output, bytes):
+            lines = output.decode('utf-8').split('\n')
+            url_dict = {}
+            for line in lines:
+                # line structure is '<label>/t<url> (fetch)'
+                split_line = line.split(' ')
+                if len(split_line) < 2:
```

Comment:
Can you explain why you chose this check?

Reply:
The last line of output is a empty line, so i use this to skip the last line.

Reply:
You can remove the last line by doing `lines = output.decode('utf-8').split('\n').pop()`